docs/world maintainer.md

```diff
@@ -0,0 +1,57 @@
+# World Maintainer
+
+A world maintainer is a person responsible for a world or part of a world in Archipelago.
+
+If a world author does not want to take on the responsibilities of a world maintainer, they can release their world as
+an unofficial [APWorld](/docs/apworld%20specification.md) or maintain their own fork instead.
+
+
+## Responsibilities
+
+Unless these are shared between multiple people, we expect the following from each world maintainer
+
+* Be on our Discord to get updates on problems with and suggestions for the world.
+* Decide if a feature (pull request) should be merged.
+* Review contents of such pull requests or organize peer reviews or post that you did not review the content.
+* Fix or point out issues when core changes break your code.
+* Use the watch function on GitHub, the #github-updates channel on Discord or check manually from time to time for new
+  pull requests. Core maintainers may also ping you if a pull request concerns your world,
+* Test (or have tested) the world on the main branch from time to time, especially during RC (release candidate) phases
+  of development.
+* Let us know of long unavailabilities.
+
+
+## Becoming a World Maintainer
+
+### Adding a World
+
+When we merge your world into the core Archipelago repository, you automatically become world maintainer unless you
+nominate someone else (i.e. there are multiple devs).
+
+### Getting Voted
+
+When a world is unmaintained, the [core maintainers](https://github.com/orgs/ArchipelagoMW/people)
+can vote for a new maintainer if there is a candidate. The majority of valid, duly votes has to be in favor.
```

Comment:
Instead of:
> The majority of valid, duly votes has to be in favor.

I suggest the phrasing:
For a vote to pass, the majority of core maintainers must vote in the affirmative.

While correct, the original phrasing is quite litigious and many will not understand it.

Reply:
To me "For a vote to pass, the majority of core maintainers must vote in the affirmative." would mean that 3 core maintainers not voting because they are awol or because they don't care would block a decision. Is this preferred?

Reply:
If there are five maintainers in total, three must vote in the affirmative to achieve a majority. It's a majority of all existing maintainers, not just the ones who vote. I do see your point about a maintainer wanting to abstain. I think I'm okay with the idea that a majority consists of 51% or more of maintainers who voted.

Reply:
Yes, I think not everyone voting could be common depending on current involvement with the world in question.

If you have a suggestion how to word that better, I'd be happy to update it.

I think i incorporate all remaining suggestions in the last commit.

Reply:
How about:
For a vote to pass, the majority of participating core maintainers must vote in the affirmative.

Reply:
Sounds good. done.